free-2.md

```diff
@@ -11,34 +11,55 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This language would serve latex users, helping them figure out how to do what they want mathematically in latex's language. 
+
+Latex is hard to learn. There are a ton of commands that you have to google search to be able to write, and you usually have to do this one at a time. Despite this, its still the most widely used math typesetting language that I know of. My idea is for a DSL that lets someone input the mathematical words for the symbols they want to use, and have the language output the latex commands for whatever they said. A language is good for this, as it would essentially be simplified google searching, which is normally what people do (e.g. "floor symbol in latex" would be someone searching for "\lfoor" and "\rfloor"). Thus, a language seems like the natural extension of what this is. 
+
+Additionally, this process only works on one symbol at a time. In my project, you could do something like:
+
+floor(n over 2)
+
+and get $\lfloor \frac{n}{2} \rfloor$ as a result.
+
```

Comment:
I'm currently learning LateX for cs81 and something like this would be super convenient to have.